### PR TITLE
Rails 7.1, Ruby 3.1, and Ruby 3.2

### PR DESCRIPTION
**Pull request by smudge**
This adds Rails 7.1, Ruby 3.1, and Ruby 3.2 to the build matrix.

The gem already works with these versions, but now they are formally tested. (As such, I bumped the gem by a patch version, to signify that this is the first verified-compatible release.)

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -12,16 +12,27 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [2.6, 2.7, 3.0]
+        ruby: ['2.6', '2.7', '3.0', '3.1', '3.2']
```
Comment (smudge) [inline .github/workflows/tests.yml:15]: I'd like to drop Ruby < 3 support in the next major release.

**Review by samandmoore — APPROVED** [review]
domainlgtm

**Review by samandmoore — APPROVED** [review]
platformlgtm too

when do you think you'll wanna do the major release to cut old versions?

File: .github/workflows/tests.yml (hunk 1 above)
Comment (samandmoore) [reply .github/workflows/tests.yml:15]: what about unsupported rails versions? those too?

Comment (smudge) [reply .github/workflows/tests.yml:15]: Yes, those too I think.

**Comment by smudge** [comment]
@samandmoore I have a branch where I've been experimenting with more fully decoupling this gem from rails, so that it can be used more readily in standalone Ruby apps/gems. That's what I was targeting as the next major release.

**Pull request merged**